app/src/main/assets/migrations/27.sql

```diff
@@ -1,3 +0,0 @@
-ALTER TABLE TransmitterData ADD COLUMN filtered_data DOUBLE;
```

Comment:
This line does need to be in here I believe, we need to make sure at least one of these migrations adds filterd data to the TransmitterData table, I dont mind if you keep it in this file or add it into migration 32, either would be just fine!